feat: improve paid actors docs for devs (#723)
apify/apify-docs: sources/academy/platform/get_most_of_actors/monetizing_your_actor.md

You can read more about Actors in store and different pricing models from the perspective of your users in our [Store documentation](https://docs.apify.com/platform/actors/running/actors-in-store).

#### Best practices for pay-per-result Actors
1. Set `minMemoryMbytes` and `maxMemoryMBytes` in [`actor.json`](https://docs.apify.com/platform/actors/development/actor-definition/actor-json) in your Actor folder. If users use more memory than you counted with in your price estimations, their runs will generate more underlying platform usage and will cost your more, which might affect your profit margin.

2. Always check `ACTOR_MAX_PAID_DATASET_ITEMS` environment variable in your Actor code. Your Actor should never return more than `ACTOR_MAX_PAID_DATASET_ITEMS` results, because the user will be not charged for more and you could start generating losses. This variable is configurable by the user to limit how much the run will cost them.

> 3. To set your price per 1,000 results, we recommend running your Actor several times to see the average cost pre 1,000 runs that is available in the Apify Console when you open run detail. Usually runs returning only 1 result are much more expensive in the underlying platform costs than runs returning many results, due to fixed costs at Actor startups. Hence, try runs with different number of results in your experiments to get the best idea about the average costs. Also make sure your runs never return 0 results, e.g. when scraping social network profiles without any videos, return at least the basic profile info.

Reviewer: ```suggestion
3. To set your price per 1,000 results, we recommend running your Actor several times to see the average cost per 1,000 runs that is available in the Apify Console when you open run detail. Usually runs returning only 1 result are much more expensive in the underlying platform costs than runs returning many results, due to fixed costs at Actor startups. Hence, try runs with different numbers of results in your experiments to get the best idea about the average costs. Also make sure your runs never return 0 results, e.g. when scraping social network profiles without any videos, return at least the basic profile info.
```

### Setting up monetization {#setting-up-monetization}

First, go to your Actor page, [**Actors → My Actors**](https://console.apify.com/actors?tab=my) and click on the Actor you want to monetize. While on your Actor page, go to the **Publication** tab on the right side of the screen into **Publication settings** and open the **Monetization** section.

![Monetization section](./images/monetization_section.png)

> If you haven't already, you will need to fill in the billing and payments details that will be used for paying out the profits. You can do that by following the link shown on the screen. Once you have the details set up, you will be able to access details on your payouts in the [**Settings > Payouts**](https://console.apify.com/account/payouts) section.

Reviewer: ```suggestion
If you haven't already, you will need to fill in the billing and payment details that will be used for paying out the profits. You can do that by following the link shown on the screen. Once you have the details set up, you will be able to access details on your payouts in the [**Settings > Payouts**](https://console.apify.com/account/payouts) section.
```

![Set up monetization](./images/monetize_actor_set_up_monetization.png)

Once you have your details setup, you can click the **Set up monetization** button that will lead you through a wizard where you will choose the monetization model and any respective fees.

![Monetization wizard](./images/monetization_wizard.png)

### Changing monetization {#changing-monetization}

You can always change the monetization settings of your Actor. Note that any changes made to an already published Actor will always take 14 days to come into effect, to give users of your Actor, many of which might be using it in their critical business flows, time to prepare for the change. To change the monetization, use the same wizard as for the setup, found in Monetization section of the Actor Publication tab.

Also note that your are allowed to adjust the monetization settings of the Actor only once per month. Please refer to our [Terms & Conditions](https://apify.com/store-terms-and-conditions) for further details and guidelines.

### How you will be paid out {#payouts}

> Payout invoices are created automatically at the beginning of the new month. You will be notified and can access the new payout invoice, and any historical invoices, in the [**Settings > Payouts**](https://console.apify.com/account/payouts) section in Apify Console. Once the new invoice is ready, you will need to review and approve it in order the payment to be processed. Note that if you do not approve the invoice, or let us know of any discrepancies, by 9th of the month, the invoice will be auto-approved by the system on 10th.

Reviewer: ```suggestion
Payout invoices are created automatically at the beginning of the new month. You will be notified and can access the new payout invoice, and any historical invoices, in the [**Settings > Payouts**](https://console.apify.com/account/payouts) section in Apify Console. Once the new invoice is ready, you will need to review and approve it in order for the payment to be processed. Note that if you do not approve the invoice, or let us know of any discrepancies, by the 9th of the month, the invoice will be auto-approved by the system on the 10th.
```
